src/backend/base/langflow/components/logic/conditional_router.py

```diff
@@ -83,18 +87,22 @@ def evaluate_condition(self, input_text: str, match_text: str, operator: str, *,
             return input_text.startswith(match_text)
         if operator == "ends with":
             return input_text.endswith(match_text)
+        if operator == "matches regex":
```

Comment:
can these be elif statements instead? functionally should operate the same, but i think makes it a little more clear to someone reading the code.